rootfs/etc/teleport/teleport-auth.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+gomplate -f /etc/teleport/teleport-auth.yaml
```

Comment:
I don't think this does anything. There's no output file. I don't think it will modify a file in place.  Even if it did, we should output it somewhere else (e.g. in this case `/host/...`)